Multisig/UI/Transaction/InitiateTransaction/ReviewSafeTransactionViewController/ReviewSafeTransactionViewController.swift

```diff
@@ -149,19 +149,20 @@ class ReviewSafeTransactionViewController: UIViewController {
                     self.safeTxGas = UInt256String(estimatedSafeTxGas)
                 }
 
-                if self.shouldLoadTransactionPreview {
+                if self.shouldLoadTransactionPreview, let transaction = self.createTransaction() {
```

Comment:
check the beginning of this method (line 113) it contains the same check, is it necessary to do it again ?

Reply:
yes because the safeTxGas and safeNonce have changed in the lines above, and the transaction is outdated now

Reply:
however, those parameters seem irrelevant for the preview endpoint, but still, I was trying to debug it and found that it is using not updated values after estimation